gthumb-appdata.patch

```diff
@@ -0,0 +1,68 @@
+--- gthumb-3.12.2/data/appdata/org.gnome.gThumb.appdata.xml.in	2022-04-03 14:38:59.000000000 -0400
++++ gthumb-3.12.2/data/appdata/org.gnome.gThumb.appdata.xml	2022-04-09 20:59:35.776975149 -0400
+@@ -1,12 +1,11 @@
+-<?xml version="1.0" encoding="UTF-8"?>
+-<!-- Copyright 2014-2017 Paolo Bacchilega <paobac@src.gnome.org> -->
++<?xml version="1.0" encoding="utf-8"?>
+ <component type="desktop-application">
+   <id>org.gnome.gThumb.desktop</id>
+-  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
It would be interesting to reduce the diff spam, many properties are just being moved from one place to the other.

Reply:
this is what appstreamcli generated using the method in the `meson.build` files, except I limited to 1 instead of 6 releases.

the proper solution is to have upstream generated it for the tarball.

Reply:
MR submitted upstream: https://gitlab.gnome.org/GNOME/gthumb/-/merge_requests/35

Reply:
I would merge this PR. Above MR was merged so the next release will be "cleaner".